### PR TITLE
build: migrating to moonrepo

### PR DESCRIPTION
**Pull request by bclark-p44**
Closes # 

## 📝 Description

> Add a brief description

## Screenshots

> Please provide screenshots for any visual changes

## Merge checklist

- [ ] Added/updated tests
- [ ] Added changeset

### CODE REVIEW

File: apps/docs/src/components/ActionCard.tsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
   href: string;
   src: string;
   title: string;
-  onClick?(): void;
+  onClick?: () => void;
 }
 
-function ActionCard(props: ActionCardProps) {
-  const { description, href, src, title } = props;
+function ActionCard(properties: ActionCardProperties) {
```
Comment (Ashley-p44) [inline apps/docs/src/components/ActionCard.tsx:14]: why the change from props?

Comment (bclark-p44) [reply apps/docs/src/components/ActionCard.tsx:14]: ugh leftover from a wrong ESLint configuration. will fix.

File: packages/react-popover/tests/Popover.test.tsx [hunk 2]
```diff
@@ -8,34 +8,34 @@ import {
...
   usePopover,
-  UsePopoverProps,
+  UsePopoverProps as UsePopoverProperties,
   usePopoverState,
-  UsePopoverStateProps,
+  UsePopoverStateProps as UsePopoverStateProperties,
 } from '../src';
 
-function Component(props: PopoverProps & UsePopoverProps & UsePopoverStateProps) {
+function Component(properties: PopoverProperties & UsePopoverProperties & UsePopoverStateProperties) {
```
Comment (Ashley-p44) [inline packages/react-popover/tests/Popover.test.tsx:18]: looks like some didn't change back to props

File: packages/react/src/utils/filter.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { ListCollection } from '@react-stately/list';
 // Will be fixed when this is moved into its own package
```
Comment (Ashley-p44) [inline packages/react/src/utils/filter.ts:2]: Do you still need this comment?

File: packages/react/src/components/CalendarRange/index.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
 export * from './CalendarRange';
 // Will be fixed when this is moved into its own package
```
Comment (Ashley-p44) [inline packages/react/src/components/CalendarRange/index.ts:2]: and here

File: packages/react-utils/src/types.ts [hunk 5]
```diff
@@ -1,9 +1,9 @@
 export type Callback<T extends unknown[]> = ((...args: T) => void) | undefined;
 
 // Cloning the types from react-aria, need to preserve the types for the mergeProps function to be typed correctly
```
Comment (Ashley-p44) [inline packages/react-utils/src/types.ts:3]: Do you still need this comment?

Comment (bclark-p44) [reply packages/react-utils/src/types.ts:3]: probably not

**Review by Ashley-p44 — APPROVED** [review]
Left a couple comments about some left over comments. Other than that, this looks great!
